Packs/GoogleSheets/Integrations/GoogleSheets/GoogleSheets.py

```diff
@@ -205,7 +205,10 @@ def make_markdown_matrix(sheets: list) -> str:
                 markdown += '|'
                 values = row.get('values')
                 for value in values:
-                    markdown += value + ' |'
+                    if not value:
+                        markdown += "" + ' |'
+                    else:
+                        markdown += value + ' |'
```

Comment:
```suggestion
                    markdown += (value or "") + ' |'
```
hope my indentation is correct, but this is cleaner imo than the condition :)